client/cmd/import.go

```diff
@@ -1,7 +1,8 @@
 package cmd
 
 import (
-	"alda.io/client/code-generator"
+	"alda.io/client/formatter"
+	"alda.io/client/generator"
```

Comment:
I think I'd prefer that we call these `code_formatter` and `code_generator`, so that the package names make it very obvious what they do.

Reply:
Yeah `generator` especially is a little ambiguous, I agree.